metrics/error-report/index.js

```diff
@@ -0,0 +1,69 @@
+/* eslint-disable import/prefer-default-export */
+/* eslint-disable no-prototype-builtins */
+import AES from "crypto-js/aes";
+import Utf8 from "crypto-js/enc-utf8";
+import ScriptLoader from "../../integrations/ScriptLoader";
+// import RudderApp from "../../utils/RudderApp";
+
+const metaData = {
+  SDK: {
+    name: "RudderStack JavaScript SDK",
+    version: "process.package_version",
+    installType: process.browser ? "CDN" : "NPM",
+  },
+};
+
+const credentials = {
+  apiKey:
+    "U2FsdGVkX182Myeej4DKayK1qKvTaxfa/9g4oz/kMD1hkuPCBKmyrWZB74x3N0qf6xuPtH8QRq4YRnIh15AaNg==", // "0d96a60df267f4a13f808bbaa54e535c"
+  appVersion: "1.0.0",
+  releaseStage: "development",
```

Comment:
Although `releaseStage` doesn't apply to our use case, it should be set to `production`.

Reply:
done